src/controllers/message.controller.js

```diff
@@ -0,0 +1,42 @@
+const Message = require("../models/message.models");
+const SubChannel = require("../models/subChannel.models");
+const {response_500, response_400, response_200} = require("../utils/responseCodes.utils");
+
+
+exports.sendMessage = async (req, res) => {
+    try {
+        const {message, attachments, user} = req.body;
+        const subChannelId = req.params.id;
```

Comment:
take the subchannel id from req.body.subchannel._id as we will be using middleware